### DISCUSSION TITLE
Can I use the Dracula theme in my blog website's code block?

### DISCUSSION THREAD

**Discussion by ghost** [dracula/dracula-ui#156] [Q&A] [answered]
Hi there, Firstly, the Dracula theme is excellent and beautiful. Is there any way can I use the Dracula theme on my blog website code blocks, like a CSS style sheet I can use to colour the code or any other way? thank you.

**Comment by luxonauta** [comment, marked as answer]
We are happy to hear this! And yes, there are many ways and tools for this; one of the most used is [highlight.js](https://highlightjs.org/), which supports Dracula.
